_posts/2022-11-10-release-1.15.3.md

```diff
@@ -0,0 +1,185 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29500'>FLINK-29500</a>] -         InitializeOnMaster uses wrong parallelism with AdaptiveScheduler
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29509'>FLINK-29509</a>] -         Set correct subtaskId during recovery of committables
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29512'>FLINK-29512</a>] -         Align SubtaskCommittableManager checkpointId with CheckpointCommittableManagerImpl checkpointId during recovery
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29539'>FLINK-29539</a>] -         dnsPolicy in FlinkPod is not overridable 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29567'>FLINK-29567</a>] -         Revert sink output metric names from numRecordsSend back to numRecordsOut
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29613'>FLINK-29613</a>] -         Wrong message size assertion in Pulsar&#39;s batch message
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29627'>FLINK-29627</a>] -         Sink - Duplicate key exception during recover more than 1 committable.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29645'>FLINK-29645</a>] -         BatchExecutionKeyedStateBackend is using incorrect ExecutionConfig when creating serializer
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29749'>FLINK-29749</a>] -         flink info command support dynamic properties
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29803'>FLINK-29803</a>] -         Table API Scala APIs lack proper source jars
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29827'>FLINK-29827</a>] -         [Connector][AsyncSinkWriter] Checkpointed states block writer from sending records
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29927'>FLINK-29927</a>] -         AkkaUtils#getAddress may cause memory leak
+</li>
+</ul>
+        
+<h2>        Improvement
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-24906'>FLINK-24906</a>] -         Improve CSV format handling and support
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28733'>FLINK-28733</a>] -         jobmanager.sh should support dynamic properties
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28909'>FLINK-28909</a>] -         Add ribbon filter policy option in RocksDBConfiguredOptions
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29134'>FLINK-29134</a>] -         fetch metrics may cause oom(ThreadPool task pile up)
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29158'>FLINK-29158</a>] -         Fix logging in DefaultCompletedCheckpointStore
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29198'>FLINK-29198</a>] -         RetryExtension doesn&#39;t make the test fail if the retries are exhausted
```

Comment:
This is also more like a build infrastructure issue that was fixed and, therefore, doesn't need to show up in the list.